.github/workflows/pull_request.yml

```diff
@@ -8,3 +8,15 @@ jobs:
   tests:
     name: Test
     uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
+  soundness:
+    name: Soundness
+    uses: swiftlang/github-workflows/.github/workflows/soundness.yml@main
+    with:
+      api_breakage_check_enabled: true
+      broken_symlink_check_enabled: true
+      docs_check_enabled: true
+      format_check_enabled: true
+      license_header_check_enabled: false
+      license_header_check_project_name: "Swift.org"
+      shell_check_enabled: true
+      unacceptable_language_check_enabled: true
```

Comment:
All of these are enabled in the workflow by default, right? So we could omit all the `: true` lines, right?